src/patternfly/components/Tabs/tabs-item-toggle-icon.hbs

```diff
@@ -0,0 +1,7 @@
+<span class="pf-c-tabs__item-toggle-icon">
+  {{#if tabs-item-toggle-icon--IsExpanded}}
+    <i class="fas fa-angle-down fa-sm" aria-hidden="true"></i>
+  {{else}}
+    <i class="fas fa-angle-right fa-sm" aria-hidden="true"></i>
```

Comment:
This will tie the icon size to markup, so if that ever changes, we can't just push a CSS change - devs will need to update the class (and however that would be done on the react-icons side, too) which can lead to inconsistencies and is a pain to update. For core we have to consider not breaking users who don't use react, so the less style we tie to markup the better.

Also if the size of the icon is tied to a class, and we change that class, that's technically breaking as someone may have applied a style to the icon using the class that changed (`.fa-sm`).

You can target `.pf-c-tabs__item-toggle-icon` in the CSS and set the font-size to sm.

```suggestion
    <i class="fas fa-angle-right fa-sm" aria-hidden="true"></i>
```